<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_login_window</class>
 <widget class="QWidget" name="create_login_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>320</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CREATE LOGIN</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/login.png</normaloff>../Images/login.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="font">
        <font>
         <family>Segoe UI Variable Text Semibold</family>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>USERNAME:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="placeholderText">
        <string>Enter your username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_password">
       <property name="font">
        <font>
         <family>Segoe UI Variable Text Semibold</family>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PASSWORD:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter your password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_viewpass">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/eye.png</normaloff>../Images/eye.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_adduser">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_connectDB">
     <property name="text">
      <string>Connect to Database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_result">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
